vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/jws/JwsService.kt

```diff
@@ -165,23 +180,22 @@ class DefaultJwsService(private val cryptoService: CryptoService) : JwsService {
         val z = cryptoService.performKeyAgreement(ephemeralKeyPair, recipientKey, jweAlgorithm)
             .getOrElse {
                 Napier.w("No Z value from native code", it)
-                return null
+                return KmmResult.failure(it)
             }
         val kdf = prependWithAdditionalInfo(z, jweEncryption, null, null)
         val key = cryptoService.messageDigest(kdf, Digest.SHA256).getOrElse {
```

Comment:
🤔

Reply:
This is fixed per spec https://datatracker.ietf.org/doc/html/rfc7518#section-4.6.2